js/recipe/metadata.js

```diff
@@ -0,0 +1,31 @@
+import { htmlElement } from "../htmlRenderer.js"
+import { els } from "../htmlRenderer.js"
+import { render } from "../htmlRenderer.js"
+
+export function metadata(recipe) {
+  // Recipe Data
+  const tags = recipe.tags;
+  const title = recipe.name;
+  const publishedAt = new Date(recipe.datePublished).toDateString();
+  const serves = recipe.recipeYield;
+
+  const recipeParentElement = document.querySelector("article.post.single");
+
+  // TITLE
+  render(recipeParentElement, htmlElement("h1")({className: "post-title", innerHTML: title}));
+
+  // METADATA
+  const metaParentElement = render(recipeParentElement, els.div({className: "post-meta clear"}));
+
+  // Date published
+  const datePublishedElement = render(metaParentElement, els.time({innerHTML: publishedAt}));
+  datePublishedElement.setAttribute("datetime", "2020-12-16T09:09:00+00:00");
```

Comment:
Can this be injected into the `els.time({innerHTML: publishedAt})` declaration? We can have specific elements take additional parameters without adding all of those parameters to `htmlElement`explicitly in one of two ways - let `htmlElement` take arbitrary parameters that we set dynamically, or extend `els.time` to also take `datetime`. I'd vote for the second to be more explicit.

Reply:
Yes, agreed. I've added an argument to els.time 👌

Reply:
Hmm, I cannot get this to work correctly, I will ask you about it later today.